crates/lib/src/stdlib/blocks/for_block.rs

```diff
@@ -225,42 +226,6 @@ impl Renderable for For {
     }
 }
 
-#[derive(Debug, Clone, ValueView, ObjectView)]
-struct ForloopObject<'p> {
```

Comment:
I think I'd prefer these staying here and either
- marking them as `pub` and having the `mod.rs` `use` them up the chain
- make the layers of `mod`s `pub` and have the render tag access it

My reason is this is all very tightly part of being a for block and would like to have that code next to each other rather than spread out.   it just happens that `render_tag` wraps the for block behavior so it makes sense to reach in to access it.

Reply:
Yeah, this make sense. I've gone with the first option as it looks a bit cleaner in my opinion.